.github/actions/load-workflow-variables/test/test.env.yml

```diff
@@ -0,0 +1,25 @@
+- dirs:
```

Comment:
What if we have 2 or more conflicting entries? What if we have a `*` on the root directory and then we have a lower level exact match?

Reply:
Currently, the lower level would win and short circuit.